docs/dev/src/design/keys.md

```diff
@@ -2,9 +2,28 @@
 
 Document the different Keys in RisingWave.
 
+## Order Key
+
+The order key is user-specified, in SQL. For example:
+
+```sql
+create materialized view m1 as select * from t1 order by i, id;
+```
+
+The order key is `i, id`, which is the order of the columns in the `order by` clause.
```

Comment:
To be more precise, the `order_key` also specifies an `OrderType` for each column, including direction (`ASC` `DESC`) and nulls ordering.